package.json

```diff
@@ -50,7 +51,23 @@
     ]
   },
   "devDependencies": {
+    "eslint": "^7.10.0",
+    "eslint-config-prettier": "^6.12.0",
     "eslint-loader": "^4.0.2",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-react": "^7.21.3",
+    "eslint-plugin-react-hooks": "^4.1.2",
+    "husky": "^4.3.0",
```

Comment:
i believe these husky, lint-staged, and prettier need to be dependencies. When npm installing, these packages are not available and i have to run `npm i husky lint-staged prettier` on your branch before I can see the effects of pre-commit hooks

Reply:
Moved to dependencies